### PR TITLE
docs(web-modeler): add collaboration limit

### PR DESCRIPTION
**Pull request by JacobiCamunda**
## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

File: docs/self-managed/modeler/web-modeler/configuration/configuration.md [hunk 1]
```diff
@@ -44,6 +44,18 @@ Web Modeler requires an SMTP server to send notification emails to users.
 | `RESTAPI_MAIL_FROM_NAME`    | [optional]<br/>Name displayed as the sender of emails sent by Web Modeler.                             | `Camunda`                     | `Camunda`     |
 | `RESTAPI_SERVER_URL`        | URL at which users access Web Modeler in the browser (used to construct links in notification emails). | `https://modeler.example.com` | -             |
 
+#### Collaboration and invitation limits
```
Comment (christinaausley) [inline docs/self-managed/modeler/web-modeler/configuration/configuration.md:47]: I feel like this is the only point in the SM docs we mention some form of collaboration, so I've placed this content here for now.

Alternatively, if we want a SM-specific doc outlining collaboration like we do [here](https://docs.camunda.io/docs/components/modeler/web-modeler/collaboration/), we should likely create a separate issue to document this.

Comment (JacobiCamunda) [reply docs/self-managed/modeler/web-modeler/configuration/configuration.md:47]: We already shipped with `alpha2` a link to the documentation which points to the collaboration page. And I think the correct place is the collaborator page as we discussed this internally already.

CC: @wollefitz

Comment (christinaausley) [reply docs/self-managed/modeler/web-modeler/configuration/configuration.md:47]: If there has already been internal discussion on this I'm fine with keeping it as-is, but worry as this is specific to Self-Managed.

Comment (wollefitz) [reply docs/self-managed/modeler/web-modeler/configuration/configuration.md:47]: In my opinion, it doesn't make sense to mention the limits on the configuration page as it should be focused on configuration properties only. The restriction don't necessarily have to be mentioned on a "Self-Managed" collaboration page - this is more about **licensing** than it is about **collaboration**.

@JacobiCamunda What should still be mentioned is the new environment variable `CAMUNDA_LICENSE_KEY` consumed by both `webapp` and `restapi` with a link to the page explaining restrictions.

> If there has already been internal discussion on this I'm fine with keeping it as-is, but worry as this is specific to Self-Managed.

What going forward:
* we put the note about collaboration limits on the page which is referenced in the Web Modeler `8.6-alpha2` updates (ie. `docs/components/modeler/web-modeler/collaboration.md`) for the `8.6-alpha2` release
* we follow up on this topic immediately afterwards in scope of https://github.com/camunda/product-hub/issues/2286

Comment (christinaausley) [reply docs/self-managed/modeler/web-modeler/configuration/configuration.md:47]: I'll move the content over to the collaboration doc 👍

**Comment by JacobiCamunda** [comment]
@wollefitz, I adjusted the text as discussed on Friday. Please take a look so we can get it merged today.

**Comment by wollefitz** [comment]
> @wollefitz, I adjusted the text as discussed on Friday. Please take a look so we can get it merged today.

Added https://github.com/camunda/camunda-docs/pull/3858/commits/0843eb2e1b51fe7eac704a3a9de5b9543ac3a15d with my suggestion.

**Review by wollefitz — DISMISSED** [review]

**Review by christinaausley — APPROVED** [review]

**Pull request merged**